QuadrupedalRobots/ETGRL/alg/sac.py

```diff
@@ -0,0 +1,122 @@
+#   Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
Can we directly import the SAC algorithm from parl?

Reply:
Can't, it‘s a modified version.